chem/chemics_init.F

```diff
@@ -451,7 +451,7 @@ subroutine chem_init (id,chem,emis_ant,scalar,dt,bioemdt,photdt,chemdt,stepbioe,
             (config_flags%chem_opt >= 31  .AND. config_flags%chem_opt <= 34)  .OR. &
              config_flags%chem_opt == 170 .OR.  config_flags%chem_opt == 198  .OR. &
              config_flags%chem_opt == 199 .OR.  config_flags%chem_opt == 201  .OR. &
-             config_flags%chem_opt == 203 .OR.                                     &
+             config_flags%chem_opt == 203 .OR.  config_flags%chem_opt == 202  .OR. &
```

Comment:
@lukeconibear 
Luke,
I see that your mod minimizes differences. However, the rest of the chem_opt comparisons in this IF test are in numerical order. Maybe swap the location of the 202 and 203 values?

Reply:
Sure, done.

Reply:
Thanks